content/d-sgov/d-sgov-právní-akty/d-sgov-právní-akty-glosář.ttl

```diff
@@ -73,8 +73,9 @@ d-sgov-právní-akty-pojem:má-související-dokument a skos:Concept;
   skos:prefLabel "má související dokument"@cs .
 
 d-sgov-právní-akty-pojem:má-fragment-znění a skos:Concept;
+  skos:altLabel "má označení fragmentu znění právního aktu"@cs ;
   skos:inScheme d-sgov-právní-akty:glosář;
-  skos:prefLabel "má označení fragmentu znění právního aktu"@cs .
+  skos:prefLabel "má fragment znění"@cs .
```

Comment:
tohle bylo správně. Má fragment znění by mělo být altLabelem, tedy přidat altLabel, ale nehcat prefLabel jak je

Reply:
No, mě se to název moc nelíbí, ale máš pravdu, že tenhle ticket je o něčem jiném, tak jsem ti to vrátil do původního stavu.